linux_os/guide/system/permissions/files/permissions_important_account_files/file_etc_security_opasswd/bash/shared.sh

```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_sle
+
+# Create /etc/security/opasswd if needed
+# Owner group mode root.root 0600
```

Comment:
The remediation code explains itself quite clearly, so I suggest to get rid of those comments.